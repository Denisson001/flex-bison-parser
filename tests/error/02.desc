Runtime error - slice func: invalid right border