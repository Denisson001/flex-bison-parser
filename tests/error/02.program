{
    string_s = "0123456789";
    print(string_s[0 : 11]);
}