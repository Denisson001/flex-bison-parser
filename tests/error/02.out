Interpreter: runtime error - slice function: invalid borders
